data/json/itemgroups/food_service.json

```
@@ -976,5 +976,49 @@
       { "item": "stout", "prob": 10, "container-item": "keg" },
       { "item": "imperial_stout", "prob": 5, "container-item": "keg" }
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "butcher_raw_meat",
+    "items": [
+      [ "meat", 8 ],
+      [ "lunchmeat", 6 ],
+      [ "offal", 10 ],
+      [ "hotdogs_frozen", 6 ],
+      { "item": "lung", "prob": 1, "charges": 2 },
```

```suggestion
      { "item": "lung", "prob": 2, "charges": 5 },
```
You might also want to bump the probability and amount of the new offal types, "prob" so that offal items show up about as much as before and "charges" because you need 5 of one to equal a meat chunk in recipes that take both.